geopandas/tests/test_sindex.py

```diff
@@ -17,8 +17,10 @@
 import pytest
 import numpy as np
 
-if compat.USE_PYGEOS:
-    import pygeos
+if compat.USE_SHAPELY_20:
+    import shapely as geom_mod
```

Comment:
```suggestion
    import shapely as mod
```

(to match usage elsewhere)